tests/test_unit/test_validators.py

```diff
@@ -59,13 +59,47 @@ class TestValidators:
         },  # invalid input
     ]
 
+    invalid_bboxes_arrays_and_expected_log = {
+        ky: [
```

Comment:
nicely refactored, but does "ky" stand for "key"? Never seen it written like that before. Do you mind adding the "e" back in (not worth the saved space)?